chrome.go

```diff
@@ -181,59 +205,54 @@ func (req *chromeRequest) Margins(margins PageMargins) {
 	req.values[formFieldMarginRight] = fmt.Sprintf("%f%s", margins.Right, margins.Unit)
 }
 
-// Landscape sets landscape form field.
+// Set the paper orientation to landscape.
 func (req *chromeRequest) Landscape(isLandscape bool) {
 	req.values[formFieldLandscapeChrome] = strconv.FormatBool(isLandscape)
 }
 
-// PageRanges sets pageRanges form field.
+// Page ranges to print, e.g., '1-5, 8, 11-13' - empty means all pages.
 func (req *chromeRequest) PageRanges(ranges string) {
-	req.values[formFieldPageRanges] = ranges
-}
-
-// GoogleChromeRpccBufferSize sets googleChromeRpccBufferSize form field.
-func (req *chromeRequest) GoogleChromeRpccBufferSize(bufferSize int64) {
-	req.values[formFieldGoogleChromeRpccBufferSize] = strconv.FormatInt(bufferSize, 10)
+	req.values[formFieldNativePageRanges] = ranges
 }
 
-// Scale sets scale form field.
+// The scale of the page rendering.
+// Default is 1.0.
 func (req *chromeRequest) Scale(scaleFactor float64) {
 	req.values[formFieldScale] = fmt.Sprintf("%f", scaleFactor)
 }
 
-// SkipNetworkIdleEvent sets skipNetworkIdleEvent form field as true.
+// Do not wait for Chromium network to be idle.
 func (req *chromeRequest) SkipNetworkIdleEvent() {
-	req.values[formFieldSkipNetworkIdleEvent] = "true"
+	req.values[formFieldSkipNetworkIdleEvent] = strconv.FormatBool(true)
```

Comment:
Oh, the one other change was just to change these from static strings to `strconv.FormatBool(true)`. This is kind of pointless, but it seems like a more strict approach to define them like this, as there is no room for mistyping the word "true" haha.

Reply:
I agree, this looks much better